xbmc/addons/Scraper.cpp

```diff
@@ -800,10 +819,12 @@ static bool PythonDetails(const std::string &ID,
                           const std::string &key,
                           const std::string &url,
                           const std::string &action,
+                          const std::string &pathSettings,
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
Is it really the intention to use a different formatting style on the newly added parameter than for all the other parameters?

Reply:
You could also adjust the style of the already existing parameters to match the current code guidelines.

Reply:
Which has absolutely nothing to do with this PR.